.rubocop.yml

```diff
@@ -3,3 +3,4 @@ Metrics/LineLength:
 
 Metrics/BlockLength:
   Max: 100
+
```

Comment:
@ahenroid Ahh sorry Andy didnt notice that. Yes Ill remove that now. cheers Mark.